guides/v2.1/frontend-dev-guide/translations/theme_dictionary.md

```diff
@@ -21,6 +21,14 @@ When the {% glossarytooltip 05099dbb-d491-4e33-a065-16035cb2d4d9 %}locale{% endg
 
 If there are different translations for one string, the theme dictionary translations have priority over the {% glossarytooltip c1e4242b-1f1a-44c3-9d72-1d5b1435e142 %}module{% endglossarytooltip %} translations, and child theme translations have priority over parent theme translations.
 
+## What is database translation
```

Comment:
```suggestion
## What are database translations
```

Reply:
Are database translations inline translations?

Reply:
We're missing some information here:
1. How do you add translations to a database?
2. Why would you want to store translations in the database as opposed to the other methods described elsewhere?